### ISSUE TITLE
How to report bitcoin wallet fraud  ? 

### ISSUE THREAD

**Issue by saeiddolat** [bitcoin-dot-org/Bitcoin.org#3530]
SCAMMER BRD
I realized BRD wallet team scammer strategy which suggested by bitcoin. Org web site ?!!!! its not like hack or profiteers person , they lead people who want to close their account , Due to the loss of their money or the long transfer time and change of account balance after sending complaints and unsolicited emails many times,dose from the main website to another scammer team in the telegram and get personal information account for close account bu benefiter pearsons . Then, they stole Balance of accounts and finally if u ask orginal web site there is no body to answer and 1% if they will answer , will tell people they are not in touch with us , but I recorded from all this step of directing you from orginal to fake web and lead to telegram which there is lot of benefiter and admin of that telegram group , and if u be lucky can see text message  from BRD  team in this telegram group which , they noticed we are closing  this group as soon as possible and wrote new address but this message is from 4 months ago, !!!!! if BRD team wanted  to close that telegram group why didnt close until now? 
Why Didnt answer support.brd.com email 
Why they dont have any support option? 
Because they are SCAM ,FRUAD 
BRD team lead peaple from careers in tool bar to another website and advice them to more costumer support going to that telegram group because there is no other choice for answer our problem ,  when u are going to that expire telegram group , will send u some messages which they  now your problem and they are benefiter and scam admin and will say tjey accsess BRD site and will clear ur problem , and take our money by our information of wallet .  Why benefiter persons access the admin group ? And how benefiter your team Can make linke which they  will take all personal details ? 
BRD team are benefiter , scam , I have record some video , also BITCOIN.ORG. Suggest this wallet if you are new user or perfect user just suggest BRD wallet , why??

**Comment by saeiddolatabadi** [comment]
SCAMMER BRD
I realized BRD wallet team scammer strategy which suggested by bitcoin. Org web site ?!!!! its not like hack or profiteers person , they lead people who want to close their account , Due to the loss of their money or the long transfer time and change of account balance after sending complaints and unsolicited emails many times,dose from the main website to another scammer team in the telegram and get personal information account for close account bu benefiter pearsons . Then, they stole Balance of accounts and finally if u ask orginal web site there is no body to answer and 1% if they will answer , will tell people they are not in touch with us , but I recorded from all this step of directing you from orginal to fake web and lead to telegram which there is lot of benefiter and admin of that telegram group , and if u be lucky can see text message from BRD team in this telegram group which , they noticed we are closing this group as soon as possible and wrote new address but this message is from 4 months ago, !!!!! if BRD team wanted to close that telegram group why didnt close until now?
Why Didnt answer support.brd.com email
Why they dont have any support option?
Because they are SCAM ,FRUAD
BRD team lead peaple from careers in tool bar to another website and advice them to more costumer support going to that telegram group because there is no other choice for answer our problem , when u are going to that expire telegram group , will send u some messages which they now your problem and they are benefiter and scam admin and will say tjey accsess BRD site and will clear ur problem , and take our money by our information of wallet . Why benefiter persons access the admin group ? And how benefiter your team Can make linke which they will take all personal details ?
BRD team are benefiter , scam , I have record some video , also BITCOIN.ORG. Suggest this wallet if you are new user or perfect user just suggest BRD wallet , why??
